src/renderers/dom/ReactDOM.js

```diff
@@ -59,15 +59,23 @@ if (__DEV__) {
   var ExecutionEnvironment = require('ExecutionEnvironment');
   if (ExecutionEnvironment.canUseDOM && window.top === window.self) {
 
-    // If we're in Chrome, look for the devtools marker and provide a download
-    // link if not installed.
-    if (navigator.userAgent.indexOf('Chrome') > -1) {
-      if (typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+    // First check if devtools is not installed
+    if (typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+      // If we're in Chrome or Firefox, provide a download link if not installed.
+      if (navigator.userAgent.indexOf('Chrome') > -1 &&
+          navigator.userAgent.indexOf('Edge') === -1) {
         console.debug(
           'Download the React DevTools for a better development experience: ' +
           'https://fb.me/react-devtools'
         );
       }
+
+      if (navigator.userAgent.indexOf('Firefox') > -1) {
+        console.debug(
+          'Download the React DevTools for a better development experience: ' +
+          'https://github.com/facebook/react-devtools/releases'
```

Comment:
We need to set up the links on our end, then they'll work :). We would make the firefox one point to AMO just like the chrome one would point continue point at the web store. The goal was to keep them short and similar and under our control so we could change the destination if we needed (same reason all of our warning links also use fb.me)